src/Library/demos/Frame/main.blp

```diff
@@ -14,20 +14,144 @@ Adw.StatusPage {
       halign: center;
       spacing: 18;
       margin-bottom: 18;
+      homogeneous: false;
+
+      Picture without_frame {
+        width-request: 180;
+        height-request: 180;
+      }
+
+      Separator {
+        margin-start: 6;
+        margin-end: 6;
+      }
+
+      Frame {
+        Picture with_frame {
+          width-request: 180;
+          height-request: 180;
+        }
+      }
+    }
+
+    Box {
+      orientation: horizontal;
       homogeneous: true;
+      margin-bottom: 24;
+
+      Label {
+        label: _("Without Frame");
+        styles ["title-4"]
+      }
 
       Label {
-        wrap: true;
+        label: _("With Frame");
+        styles ["title-4"]
+      }
+    }
+
+    Separator {
+      margin-bottom: 24;
+    }
+
+    Label {
+      label: _("Without Frame");
+      margin-bottom: 12;
+      halign: start;
+      styles ["title-4"]
+    }
+
+    Box {
+      margin-bottom: 18;
+      halign: center;
+      styles ["toolbar"]
+
+      Button {
+        icon-name: "plus";
+      }
+
+      Button {
+        icon-name: "minus";
+      }
+
+      Button {
+        icon-name: "bell";
+      }
+    }
+
+    Label {
+      halign: start;
+      label: _("With Frame");
+      margin-bottom: 12;
+      styles ["title-4"]
+    }
+
+
+    Frame {
+      margin-bottom: 24;
+      Box {
+        halign: center;
+        styles ["toolbar"]
+
+        Button {
+          icon-name: "plus";
+        }
+
+        Button {
+          icon-name: "minus";
+        }
+
+        Button {
+          icon-name: "bell";
+        }
+      }
+    }
+
+    Separator {
+      margin-bottom: 24;
+    }
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+      spacing: 18;
+      margin-bottom: 18;
+      homogeneous: false;
+
+      ScrolledWindow {
+        height-request: 240;
+        width-request: 240;
+
+      TextView textview_with_frame {
```

Comment:
It looks like these two ID's got mixed up, since this is the TextView without a frame.

I think you're right about the scrolled window here though, since it has more to do with `Gtk.ScrolledWindow` than `Gtk.TextView`. Just bump in the indentation here so the formatting is correct.